tutorials/classify_maizeland_NG

```diff
@@ -0,0 +1,234 @@
+---
+title: Rapid classification of croplands: Case Study of smallholder maize-cultivated lands
+description: A short description of the tutorial, all on one line with no carriage returns.
+author: PJNation
+tags: maize, smallholder, cropland, classify, rapid, binary
+date_published: 2020-06-31
+---
+#
+
+<!--
+Copyright 2019 The Google Earth Engine Community Authors
```

Comment:
Please update copyright year before submitting.

Reply:
Indeed...done; I'm making some checks prior to submitting.